### ISSUE TITLE
nbinom1 vs nbinom2 parameterisation

### ISSUE THREAD

**Issue by jeffreypullin** [HelenaLC/muscat#58]
Hi, 

Firstly, thanks for such a great paper and associated package!

I have a small question about the parameterisation of the negative binomial distribution used when fitting the 'full' mixture model - MM-nbinom in the paper.

Specifically: why was the `nbinom1` parameterisation used instead of `nbinom2` which is what I believe edgeR and the `rnbinom` function uses?

I little bit of testing I've done suggests that the difference between them may be small in practice, but I was just curious about why `nbinom1` was preferred.

Thanks, 
Jeffrey

**Comment by plger** [comment]
Hi Jeffrey,
we tried both, and to be honest the decisive factor was that we had a lot of trouble finding a stable (and computed within a decent amount of time) implementation of nbinom2 mixed models (we had fit errors on a large proportion of genes). Since we had observed that, for the genes fitted, there was a good correlation between the DEA -log10(p-values) of the two, we therefore chose nbinom1. You are right however that this is different to what edgeR does.
